test/e2e/policies.test-suite/memtier/n4c16/test04-available-resources/code.var.sh

```diff
@@ -0,0 +1,46 @@
+# Test that AvailableResources are honored.
+
+# Test explicit cpuset in AvailableResources.CPU
+terminate cri-resmgr
+AVAILABLE_CPU="cpuset:4-7,8-10"
+cri_resmgr_cfg=$(instantiate cri-resmgr-available-resources.cfg)
+launch cri-resmgr
+
+# pod0: exclusive CPUs
+CPU=3 create guaranteed
+verify "cpus['pod0c0'] == {'cpu04', 'cpu05', 'cpu06'}" \
+       "mems['pod0c0'] == {'node1'}"
+
+# pod1: shared CPUs
+CONTCOUNT=2 CPU=980m create guaranteed
+verify "cpus['pod1c0'] == {'cpu08', 'cpu09', 'cpu10'}" \
+       "cpus['pod1c1'] == {'cpu08', 'cpu09', 'cpu10'}" \
+       "mems['pod1c0'] == {'node2'}" \
+       "mems['pod1c1'] == {'node2'}"
+kubectl delete pods --all --now
+reset counters
+
+# Test cgroup cpuset directory in AvailableResources.CPU
+terminate cri-resmgr
+CRIRM_CGROUP=/sys/fs/cgroup/cpuset/cri-resmgr-test-05
+vm-command "rm -rf $CRIRM_CGROUP; mkdir $CRIRM_CGROUP; echo 1-4 > $CRIRM_CGROUP/cpuset.cpus"
+AVAILABLE_CPU="\"$CRIRM_CGROUP\""
```

Comment:
Could this test be also repeated with `AVAILABLE_CPU="\"${CRIRM_CGROUP#/sys/fs/cgroup/cpuset}"\"` ? It should produce identical results and that's the kubelet-compatible way of providing the path.

Reply:
Otherwise it LGTM.

Reply:
Thanks! Test added.